test/targets/create-target.e2e-spec.ts

```diff
@@ -77,8 +78,21 @@ describe('POST /targets', () => {
       })
 
       it('should return the newly created target', async () => {
-        const target = await targets.last()
-        expect(response.body).toEqual(TargetDto.from(target))
+        firstTarget = await targets.last()
+        expect(response.body.target).toEqual(TargetDto.from(firstTarget))
+      })
+      
+      describe('when there is another target nearby with the same topic', () => {
+        it('should create a new match', async () => {
+          response = await request(app.getHttpServer())
+            .post('/targets')
+            .set('Authorization', `Bearer ${accessToken}`)
+            .send({ ...mockTarget2, topicId: mockTopic.id })
+            .expect('Content-Type', /json/)
+            .expect(201)
+          firstTarget.topic = undefined // Matches in the response don't include a topic
+          expect(response.body.matches).toEqual(TargetDto.fromArray([firstTarget]))
```

Comment:
You are using the target from the previous test here, which means that you are relying on the current state and this makes your tests dependent which is not something that you want since it'll make your test suit brittle. The database should be clean between test runs to avoid this